src/AppBundle/Entity/User.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Entity\Repositories\UserRepository")
+ * @ORM\Table(name="users")
+ */
+class User implements UserInterface
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $username;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $password;
+
+    /**
+     * Get id
+     *
+     * @return integer
+     */
+    public function getId() : int
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set username
+     *
+     * @param string $username
+     *
+     * @return self
+     */
+    public function setUsername(?string $username) : self
```

Comment:
username is not nullable and we can write:
`public function setUsername(string $username) : self`
also for password